job-center/instructions.md

```diff
@@ -0,0 +1,38 @@
+# POST A JOB
+
+The KubeEdge job center aims to provide a platform for community partners and developers to exchange job opportunities. Partners can post job openings here and provide detailed job information and requirements, while community users can select job positions based on their experience and interests. We sincerely hope that you can find suitable jobs or excellent talents here.
+
+This guide is intended to help you understand how to post your job positions in the community job center and provide guidelines for writing job information documents.
+
+# Requirement for Posting Jobs
+
+- You must be a partner of KubeEdge community, including but not limited to having used KubeEdge in your company's products or solutions, contributing to the community, and submitting your use cases.
```

Comment:
We have not yet defined the "partner" role or relationship of an organization to the community, may call it "contributor or adopter"?

```suggestion
- You must be a contributing organization or adopter of the KubeEdge community, including but not limited to having used KubeEdge in your company's public products or public solutions, contributing to the community, and submitting your use cases.
```